books/free-programming-books-id.md

```diff
@@ -108,6 +108,9 @@
 * [Dokumentasi Vue Bahasa Indonesia](https://github.com/vuejs-id/docs)
 
 
+### Machine Learning
+* [Belajar Machine Learning dengan Python Scikit-Learn](https://www.youtube.com/playlist?list=PL2O3HdJI4voHNEv59SdXKRQVRZAFmwN9E)
+
```

Comment:
Solving linter issues, adding more info about resource like right title and instructor...

```suggestion
### Machine Learning

* [Tutorial Belajar Machine Learning Dasar \| Python Scikit-Learn](https://www.youtube.com/playlist?list=PL2O3HdJI4voHNEv59SdXKRQVRZAFmwN9E) - 


```